faqs/galaxy/visualisations_igb.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Open History files in Integrated Genome Browser (IGB)
+area: visualisation
+box_type: tip
+layout: faq
+contributors: [paige-kulzer]
+---
+
+You can open some file types in Integrated Genome Browser (IGB), a desktop genome browser. ([Supported File Types](https://wiki.bioviz.org/confluence/display/igbman/File+Formats))
+
+Here's how:
+
+1. Install IGB on your computer ([download page](https://bioviz.org/download.html)).
+2. Start IGB.
+3. In Galaxy, click the desired dataset's name to expand it.
+   - Check that the reference genome (dbkey) is set
+   ([instructions](https://training.galaxyproject.org/training-material/faqs/galaxy/datasets_change_dbkey.html))
```

Comment:
please use the link function for internal links, like here for example: https://github.com/galaxyproject/training-material/blob/c4ad059255fb949d6cf861d96de87858743016ea/faqs/gtn/gtn_event_create.md?plain=1#L10